docs_user/modules/openstack-tls_everywhere.adoc

```diff
@@ -0,0 +1,199 @@
+[id="tls-everywhere_{context}"]
+
+//:context: tls
+
+//kgilliga: This module will be converted to an assembly. Check xref contexts.
+
+= TLS Everywhere
+
+_Disclaimer: the below steps were reproduced on a FreeIPA 4.10.1 server. The location of files and directories may slightly change on different versions._
+
+== Prerequisites
+
+* Check that the source deployment is using TLS Everywhere, skip this step if it's not.
+* Make sure the previous Adoption steps have been performed successfully.
+* Make sure the backend services on the new deployment are not started yet.
+
+== Variables
+
+Define the shell variables used in the following steps. The values are illustrative and refer to a single node standalone director deployment. Use values that are correct for your environment:
+
+IPA_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.100 podman exec -ti freeipa-server-container"
+
+In this example the FreeIPA instance is running on a separate host, in a container.
```

Comment:
@jistr I'm curious what you think? This is currently written in a way so that we can execute the steps manually on our test environment

Reply:
Yea Ollie has a point but at the moment we have quite a lot of CRC-isms in the examples everywhere. Maybe we should be adding `ifeval`s for these examples as well to split them between upstream and downstream variants [1]. But i wouldn't block this patch on it.

[1] https://github.com/openstack-k8s-operators/data-plane-adoption/blob/d3951b133ba112911e6a6e98afd1e44b8c167d31/docs_user/modules/openstack-mariadb_copy.adoc?plain=1#L50

Reply:
Fabricio created a lint for this https://github.com/openstack-k8s-operators/data-plane-adoption/pull/368 so we should indeed split this example into upstream (dev focused with install yamls) vs. downstream (user focused) value, otherwise it will fail the lint.

Reply:
+1 on using real host for customer docs.